questionnaire-ko.json

```diff
@@ -1,434 +1,434 @@
+          "question": "오픈소스 컴플라이언스 문의를 받고 대응하기 위한 책임을 부여하는 문서화된 절차가 있습니까?",
+          "number": "2.d",
+          "type": "YES_NO",
           "specReference": [
-            "2.1",
-            "2.2.1"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Have the identified Program roles been properly staffed and adequately funded?",
-          "number": "2.e",
-          "type": "YES_NO",
+            "2.1",
+            "2.2.1"
+          ]
+        },
+        {
+          "correctAnswer": "네",
+          "question": "확인된 프로그램 역할을 지원하는 사람, 그룹 또는 기능을 문서화했습니까?",
+          "number": "2.e",
+          "type": "YES_NO",
           "specReference": [
-            "2.1",
-            "2.2.2"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Has legal expertise to address internal and external open source compliance been identified?",
-          "number": "2.f",
-          "type": "YES_NO",
+            "2.1",
+            "2.2.2"
+          ]
+        },
+        {
+          "correctAnswer": "네",
+          "question": "확인된 프로그램 역할에 적절하게 인원이 충원되었고 적합한 자금이 제공되었습니까?",
+          "number": "2.f",
+          "type": "YES_NO",
           "specReference": [
-            "2.1",
-            "2.2.3"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a documented procedure assigning internal responsibilities for open source compliance?",
-          "number": "2.g",
-          "type": "YES_NO",
+            "2.1",
+            "2.2.3"
+          ]
+        },
+        {
+          "correctAnswer": "네",
+          "question": "오픈소스 컴플라이언스와 관련된 내부 또는 외부의 법률 전문 지식을 이용할 수 있는 방법이 확인되었습니까?",
+          "number": "2.g",
+          "type": "YES_NO",
           "specReference": [
-            "2.1",
-            "2.2.4"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a documented procedure for handling review and remediation of non-compliant cases?",
-          "number": "2.h",
-          "type": "YES_NO",
+            "2.1",
+            "2.2.4"
+          ]
+        },
+        {
+          "correctAnswer": "네",
+          "question": "오픈소스 컴플라이언스에 대한 내부 책임을 할당하는 문서화된 절차가 있습니까?",
+          "number": "2.h",
+          "type": "YES_NO",
           "specReference": [
-            "2.1",
-            "2.2.5"
-          ]
-        }
-      ]
-    },
-    {
-      "name": "Part 3",
-      "title": "3 Open source content review and approval",
-      "questions": [
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a documented procedure for identifying, tracking and archiving information about the open source components in a Supplied Software release?",
-          "number": "3.a",
-          "type": "YES_NO",
+            "2.1",
+            "2.2.5"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Yes",

```

Comment:
This looks incorrect - should be "Part 3"

Reply:
PR later.